### ISSUE TITLE
Top-level enums missing from generated .d.ts file

### ISSUE THREAD

**Issue by bobvawter** [dcodeIO/protobuf.js#922]
protobuf.js version: 6.8.0

When using pbts to generate a TypeScript .d.ts file, any proto enums declared in the top-level of a file (i.e. no enclosing message) are missing from the generated definition file, even though they may be referenced elsewhere within the definition file.  Enums that are enclosed within message types are generated correctly.

**Comment by gaultier** [comment]
Only happens with the flag `--es6` apparently.

**Comment by bobvawter** [comment]
Confirmed, switching to `-w commonjs` fixes the code-gen.  Thanks @gaultier.

**Comment by sergi** [comment]
Yep, confirmed here as well. Definitions generated with `--es6` omits top level enums.

**Comment by dcodeIO** [comment]
Closing this issue because it has been inactive for more than two years and the project has changed substantially.

If this is still relevant, please open a fresh issue with current reproduction steps, version details, and any updated context.

Please also consider that maintaining protobuf.js takes time and effort. If the project is valuable to you or your organization, sponsorship is appreciated.

Thanks!
